src/Illuminate/Database/DatabaseServiceProvider.php

```diff
@@ -50,6 +50,10 @@ public function register()
         $this->app->singleton('db', function ($app) {
             return new DatabaseManager($app, $app['db.factory']);
         });
+
+        $this->app->singleton('db.connection', function ($app) {
```

Comment:
Why not? This resolves default connection. If you need other you should use `DatabaseManager::connection()` don't you?